lumps/dehacked/dehacked.txt

```diff
@@ -156,43 +156,43 @@ STARTUP5========================================================================
 #else
 STARTUP5============================================================================\nThis is Freedoom, the free content first person shooter.\n\nFreedoom is freely redistributable under the terms of the modified BSD\nlicense. Check out the Freedoom website for more information:\n\n    https://freedoom.github.io/\n============================================================================
 #endif
-GOTARMOR = Put on a polaric shielding vest.
-GOTMEGA = Put on the attuned polaric energy armor.
-GOTHTHBONUS = Harvested an ectoplasmic polyp.
-GOTARMBONUS = Connected a polaric field emitter.
-GOTSTIM = Opened a roll of regenerative nanobot gauze.
-GOTMEDINEED = Nano-surgeried some really sensitive parts!
-GOTMEDIKIT = Applied a regenerative nano-surgery kit.
+GOTARMOR = Equipped a force field vest.
+GOTMEGA = Equipped the attuned force field armor.
+GOTHTHBONUS = Consumed a health boost.
+GOTARMBONUS = Equipped an armor boost.
+GOTSTIM = Consumed a small health refill.
+GOTMEDINEED = Consumed a large health refill. Not dead yet!
+GOTMEDIKIT = Consumed a large health refill.
 GOTSUPER = Ectoplasmic surge!
-GOTBLUECARD = Picked up a blue passcard.
-GOTYELWCARD = Picked up a yellow passcard.
-GOTREDCARD = Picked up a red passcard.
-GOTBLUESKUL = Picked up a blue skeleton key.
-GOTYELWSKUL = Picked up a yellow skeleton key.
-GOTREDSKULL = Picked up a red skeleton key.
+GOTBLUECARD = Secured the blue passcard!
+GOTYELWCARD = Secured the yellow passcard!
+GOTREDCARD = Secured the red passcard!
+GOTBLUESKUL = Secured the blue skeleton key!
+GOTYELWSKUL = Secured the yellow skeleton key!
+GOTREDSKULL = Secured the red skeleton key!
 GOTINVUL = Vanguard of the gods!
 GOTBERSERK = Smash them to pieces!
 GOTINVIS = Invisibility!
 GOTSUIT = Rescue operations suit.
 GOTMAP = Area survey map.
 GOTVISOR = Low-light goggles.
 GOTMSPHERE = Negentropic surge!
-GOTCLIP = Picked up an ammo clip.
-GOTCLIPBOX = Picked up a box of ammo.
-GOTROCKET = Picked up a missile.
-GOTROCKBOX = Picked up a crate of missiles.
-GOTCELL = Picked up a small energy recharge.
-GOTCELLBOX = Picked up a large energy recharge.
-GOTSHELLS = Picked up some shotgun shells.
-GOTSHELLBOX = Picked up a box of shotgun shells.
-GOTBACKPACK = You found a backpack!
-GOTBFG9000 = The SKAG 1337 ... time to kick some ass!
-GOTCHAINGUN = You got the minigun!
-GOTCHAINSAW = You got the ripsaw!
-GOTLAUNCHER = You got the missile launcher!
-GOTPLASMA = You got the polaric energy weapon!
-GOTSHOTGUN = You got the pump-action shotgun!
-GOTSHOTGUN2 = You got the double-barrelled shotgun!
+GOTCLIP = Acquired a bullet clip.
+GOTCLIPBOX = Acquired a case of bullets.
+GOTROCKET = Acquired a missile.
+GOTROCKBOX = Acquired a crate of missiles.
+GOTCELL = Acquired a small energy recharge.
+GOTCELLBOX = Acquired a large energy recharge.
+GOTSHELLS = Acquired some shotgun shells.
+GOTSHELLBOX = Acquired a box of shotgun shells.
+GOTBACKPACK = A backpack for all your ammo storage needs!
```

Comment:
How about "combat pack" instead of backpack?

Reply:
Trying to look up that term on DuckDuckGo brings up
1. a lot of video game stuff that has nothing to do with carrying devices at all
2. some actual military gear that appears to be more not-backpacks than backpacks

I think it's best to leave it as it is for now (and we've still got "shotgun" anyway).

Reply:
On further thought it might be a good idea to take a bigger look at whatever remaining stuff (shotgun, backpack, etc.) that uses the same terminology as id and see if they can or ought to be rewritten.

But as a setting/lore thing it does feel outside the scope of this already-sprawling PR though.

Reply:
I can see how "combat pack" could be intended to make it clearer that this thing's only supposed to store ammo and not health or something.

But then that just makes imagine a "combat trauma pack" that lets you carry 4 medikits for rapid deployment at any time... and is the soulsphere replacement...